build.gradle.kts

```diff
@@ -65,8 +65,8 @@ tasks {
 val configValues = mapOf(
   "dateLibrary" to "java8-localdatetime",
   "serializationLibrary" to "jackson",
-  "useBeanValidation" to "false",
-  "enumPropertyNaming" to "UPPERCASE",
+  "enumPropertyNaming" to "original",
+  "useSpringBoot3" to "true",
```

Comment:
Whilst this makes no obvious impact we should be setting these here during generation of API's for spring boot 3 projects.